modules/root/root_api.rb

```diff
@@ -12,6 +12,27 @@ class Proxy::RootApi < Sinatra::Base
     end
   end
 
+  get "/features/v2" do
```

Comment:
Just a thought if this should be `/v2/features` if addtional APIs ever changed or came online you'd either get a pattern of:

```
/v2/blah
/v2/foo
```

or

```
/foo/v2
/bar/v2
```